agenta-cli/agenta/sdk/tracing/llm_tracing.py

```diff
@@ -93,41 +91,118 @@ def client(self) -> AsyncObservabilityClient:
             base_url=self.host, api_key=self.api_key, timeout=120  # type: ignore
         ).observability
 
-    def set_span_attribute(
+    ### --- API --- ###
+
+    @debug()
+    def get_context(self):
+        tracing = tracing_context.get()
+
+        return tracing
+
+    @debug()
+    def update_baggage(
         self,
         attributes: Dict[str, Any] = {},
     ):
+        if self.baggage is None:
+            self.baggage = {}
+
+        for key, value in attributes.items():
+            self.baggage[key] = value
+
+    @debug()
+    def open_trace(
```

Comment:
What is the reason behind the renaming?

Reply:
Ooph, this happened in multiple layers. From the existing nomenclature (start / end), to context vars in other languages (open / close), to context managers in Python (enter / exit), back to OTel (start / end). The "final" nomenclature in this PR wasn't necessary an active deliberate thought.

I simply made sure that the user API was kept intact.

Ultimately, we might want to pick our reference (OTel, context managers in Python, or something else).